### PR TITLE
Add high level and sub companies api endpoints for fetching emails

### PR DESCRIPTION
**Pull request by SajedHassan**

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -3069,6 +3073,151 @@ paths:
...
+          $ref: '#/components/responses/401'
+        403:
+          $ref: '#/components/responses/403'
+        404:
+          $ref: '#/components/responses/404'
+        409:
+          $ref: '#/components/responses/409'
+        500:
+          $ref: '#/components/responses/500'
+  email_messages/mark_all_as_read:
```
Comment (dem) [inline swagger.yaml:3187]: @SajedHassan Add / at the start:
```
/email_messages/mark_all_as_read
```

**Review by dem — CHANGES_REQUESTED** [review]
@SajedHassan Small tiny commentt of fix

**Review by vladimir-konnov — APPROVED** [review]
@SajedHassan looks ok

**Review by dem — APPROVED** [review]
ok to go

**Pull request merged**